simulation/include/traccc/simulation/simulator.hpp

```diff
@@ -8,6 +8,7 @@
 #pragma once
 
 // Project include(s).
+#include "traccc/definitions/qualifiers.hpp"
```

Comment:
Is this here by mistake? :confused: There should be no device code here, should it?

Reply:
True, removed